src/HDF5Msg.chpl

```diff
@@ -1207,9 +1228,83 @@ module HDF5Msg {
                     localSegs = localSegs - startValIdx;
                     writeSegmentedComponentToHdf(file_id, group, SEGMENTED_OFFSET_NAME, localSegs);
                 }
+            }
+            // write attributes for arkouda meta info
+            writeArkoudaMetaData(file_id, group, objType, dtype);
+        }
+    }
 
-                writeArkoudaMetaData(file_id, group, objType, dtype);
+    /*
+    * Write SegArray containing Strings values to HDF5
+    * String offsets are localized to SegArray offsets. The String values are then localized to ensure entire
+    * value of the string can be read.
+    */
+    proc writeNestedSegmentedDistDset(filenames: [] string, group: string, objType: string, overwrite: bool, values, segments, st: borrowed SymTab, type t) throws {
+        // TODO - better variable naming
```

Comment:
do we still need this todo?

Reply:
No this was left over. It's been removed.